library/std/src/process.rs

```diff
@@ -1417,6 +1417,13 @@ impl From<fs::File> for Stdio {
 ///
 /// [`status`]: Command::status
 /// [`wait`]: Child::wait
+//
+// We speak slightly loosely (here and in various other places in the stdlib docs) about `exit`
+// rather than `_exit`.  Strictly speaking, an exit status is a value passed to the `_exit` system
+// call, around which `exit` is a library wrapper.  But anyone who knows the difference will not be
```

Comment:
Strictly speaking, `_exit` is not a system call; it's a thin library wrapper around the `exit` system call, as distinct from the `exit` library function which includes cleanup. If we're adding precision in this comment, let's not be loose about "system call" either.

Reply:
https://www.freebsd.org/cgi/man.cgi?query=_exit&sektion=2&n=1

```
The _exit() system call...
```

Of course, when generalising across Unices, syscall names are a matter of tradition and culture; the actual technical names vary.  On Linux, most executables exit nowadays via `exit_group` which I would argue is "an `_exit` syscall".  Despite mainly being a LInux person I'm quoting BSD here because they are typically a better guide to traditional Unix approaches.